lib/omniauth/class_link/version.rb

```diff
@@ -1,5 +1,5 @@
 module Omniauth
   module ClassLink
-    VERSION = "0.3.0"
+    VERSION = "0.3.1"
```

Comment:
Please do not change VERSION in pull requests. 
It's up to gem maintainer to bump the version.